exception.py

```diff
@@ -0,0 +1,17 @@
+"""
+Define your custom class for exceptions. Show an example how it might be used.
+"""
+
+
+class Calculator:
```

Comment:
This class should be inherited from the base class with exceptions.

Reply:
done